evalml/tests/utils_tests/test_woodwork_utils.py

```diff
@@ -203,3 +208,49 @@ def test_infer_feature_types_raises_invalid_schema_error():
         df.ww.init()
         df.drop(columns=["b"], inplace=True)
         infer_feature_types(df)
+
+
+def test_ordinal_retains_order_min():
+    features = pd.DataFrame(
+        {
+            "non-ordinal": [0, 1, 2, 3, 4, 5],
+            "ordinal": [0, 1, 2, 2, 1, 0],
+            "categorical": ["red", "white", "blue", "red", "white", "blue"],
+            "datetime": [
+                "2020-09-10",
+                "2020-09-11",
+                "2020-09-12",
+                "2020-09-13",
+                "2020-09-14",
+                "2020-09-15",
+            ],
+        }
+    )
+    logical_types = {
+        "non-ordinal": "Age",
+        "ordinal": Ordinal(order=[0, 1, 2]),
+        "categorical": Categorical(encoding="Encoding"),
+        "datetime": Datetime(datetime_format="%Y-%m-%d"),
+    }
+    features.ww.init(logical_types=logical_types)
+
```

Comment:
The testing strategy here will be to make sure that the values set in the WW init, `order` for Ordinal and `datetime` for Datetime, remain after they pass through our utility function.  With the exception of categorical, which accepts a value for its init but does nothing with it.

Reply:
This makes sense to me, but I'm a little confused about the comments in the code. For example, "    # Ordinal type doesn't retain the 'order' property" --> aren't we testing that the ordinal type *does* retain the order property? Or am I totally confused lol